spec/lib/topic_query_spec.rb

```diff
@@ -2092,4 +2092,74 @@ def suggested_for(topic)
       expect(original_topic_query.list_latest.topics.map(&:id)).to eq([topic2, topic1].map(&:id))
     end
   end
+
+  describe "precedence of categories and tag setting" do
+    fab!(:watched_category) { Fabricate(:category) }
```

Comment:
```suggestion
    fab!(:watched_category) do 
      Fabricate(:category).tap do |category|
        CategoryUser.create!(
          user: user,
          category: category,
          notification_level: CategoryUser.notification_levels[:watching],
        )
      end
    end
```

I think this makes the fabrication clearer and should apply this to the other fabrication as well.